build_tools/cmake/riscv.toolchain.cmake

```diff
@@ -16,15 +16,11 @@ if(RISCV_TOOLCHAIN_INCLUDED)
 endif(RISCV_TOOLCHAIN_INCLUDED)
 set(RISCV_TOOLCHAIN_INCLUDED true)
 
-set(CMAKE_SYSTEM_PROCESSOR riscv)
```

Comment:
is there any specific reason we're getting rid of this? According to cmake's document, we have to set it to target architecture when cross-compiling: https://cmake.org/cmake/help/latest/variable/CMAKE_SYSTEM_PROCESSOR.html